RELEASING.adoc

```diff
@@ -0,0 +1,327 @@
+2022-01-14 11:46:03.616  INFO 17 --- [pool-1-thread-2] i.s.c.r.sonatype.SonatypeService         : Deployed org/springframework/cloud/spring-cloud-starter-app-broker/1.4.0/spring-cloud-starter-app-broker-1.4.0.pom.asc
+2022-01-14 11:46:03.622  INFO 17 --- [pool-1-thread-6] i.s.c.r.sonatype.SonatypeService         : Deployed org/springframework/cloud/spring-cloud-starter-app-broker/1.4.0/spring-cloud-starter-app-broker-1.4.0.module.asc
+2022-01-14 11:46:03.634  INFO 17 --- [pool-1-thread-1] i.s.c.r.sonatype.SonatypeService         : Deployed org/springframework/cloud/spring-cloud-starter-app-broker/1.4.0/spring-cloud-starter-app-broker-1.4.0.pom
+2022-01-14 11:46:03.640  INFO 17 --- [pool-1-thread-7] i.s.c.r.sonatype.SonatypeService         : Deployed org/springframework/cloud/spring-cloud-starter-app-broker/1.4.0/spring-cloud-starter-app-broker-1.4.0.pom.md5
+2022-01-14 11:46:03.659  INFO 17 --- [pool-1-thread-4] i.s.c.r.sonatype.SonatypeService         : Deployed org/springframework/cloud/spring-cloud-starter-app-broker/1.4.0/spring-cloud-starter-app-broker-1.4.0.pom.sha1
+2022-01-14 11:46:03.686  INFO 17 --- [pool-1-thread-8] i.s.c.r.sonatype.SonatypeService         : Deployed org/springframework/cloud/spring-cloud-starter-app-broker/1.4.0/spring-cloud-starter-app-broker-1.4.0.jar
+2022-01-14 11:46:03.688  INFO 17 --- [pool-1-thread-3] i.s.c.r.sonatype.SonatypeService         : Deployed org/springframework/cloud/spring-cloud-starter-app-broker/1.4.0/spring-cloud-starter-app-broker-1.4.0.jar.md5
+2022-01-14 11:46:03.695  INFO 17 --- [pool-1-thread-5] i.s.c.r.sonatype.SonatypeService         : Deployed org/springframework/cloud/spring-cloud-starter-app-broker/1.4.0/spring-cloud-starter-app-broker-1.4.0.jar.sha1
+2022-01-14 11:46:03.698  INFO 17 --- [pool-1-thread-2] i.s.c.r.sonatype.SonatypeService         : Deployed org/springframework/cloud/spring-cloud-starter-app-broker/1.4.0/spring-cloud-starter-app-broker-1.4.0.jar.asc
+2022-01-14 11:46:03.699  INFO 17 --- [           main] i.s.c.r.sonatype.SonatypeService         : Deploy complete. Closing staging repository
+2022-01-14 11:46:03.803  INFO 17 --- [           main] i.s.c.r.sonatype.SonatypeService         : Close requested. Awaiting result
+2022-01-14 11:46:49.673  INFO 17 --- [           main] i.s.c.r.sonatype.SonatypeService         : Staging repository closed
+2022-01-14 11:46:49.763  INFO 17 --- [           main] i.s.c.r.sonatype.SonatypeService         : Staging repository released
+Sync complete
+----
+
+After that step succeeded, we can go and create a new release in GitHub adding the newly created tag and writing the release notes: https://github.com/spring-cloud/spring-cloud-app-broker/releases
+
+The version bump and the branching does not happen automatically anymore, so we need to do the following:
```

Comment:
I thought the release script bumped the version on the release tag that gets created? But you then must merge the tag back in manually.